packages/devtools_app/lib/src/timeline/timeline_model.dart

```diff
@@ -347,6 +351,10 @@ class OfflineFullTimelineData extends FullTimelineData {
   }
 }
 
+mixin OfflineData<T extends TimelineData> on TimelineData {
```

Comment:
why is this a mixin rather than an interface if it doesn't actually have a concrete implementation of any methods.

Reply:
will change to interface

Reply:
discussed offline. Leaving as mixin